### PR TITLE
fix: added displayName and __ui__ to tag components

### LINKED ISSUE #2910: `displayName` is not set for the component of "@yamada-ui/tag"
script: component-display-name
package: @yamada-ui/tag

### Description
The component "@yamada-ui/tag" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

### PR DESCRIPTION
**Pull request by 273Do**
Closes #2910

## Description

Added displayName and __ui__ to tag components.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: .changeset/eighty-mice-boil.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/tag": major
```
Comment (odendayoko) [inline .changeset/eighty-mice-boil.md:2]: @273Do 
This is a minor fix and should be changed to a patch.

**Review by odendayoko — CHANGES_REQUESTED** [review]
@273Do 
Thanks for your contribution! 

The `FC` type of this component must be imported from `@yamada-ui/core`  since `__ui__` cannot be specified in react `FC`.

Please make additional changes.

File: .changeset/eighty-mice-boil.md (hunk 1 above)
Comment (273Do) [reply .changeset/eighty-mice-boil.md:2]: Ok! 
Sorry for the many mistakes.

File: packages/components/tag/src/tag.tsx
Comment (odendayoko) [inline packages/components/tag/src/tag.tsx:1]: @273Do 
I'm sorry I forgot to tell you.

You also need to add `ui` and `displayName` to `CloseButton` and `CloseIcon` in the `Tag`.

I'm very sorry, but could you please fix it?

**Review by odendayoko — CHANGES_REQUESTED** [review]

File: packages/components/tag/src/tag.tsx
Comment (273Do) [reply packages/components/tag/src/tag.tsx:1]: Sorry, I'll fix it! Thank you for your patience💦

Comment (odendayoko) [reply packages/components/tag/src/tag.tsx:1]: Sorry if I'm not explaining it well.

Please set after each component definition like this.

https://github.com/yamada-ui/yamada-ui/pull/2989

Add Tag under the `Tag` definition , CloseIcon under `CloseIcon` definition , and CloseButton under `CloseButton` definition .

File: .changeset/polite-cobras-type.md
[the remark this answers is not in the document]
Comment (273Do) [reply .changeset/polite-cobras-type.md:1]: Sorry...

**Review by odendayoko — APPROVED** [review]
@273Do 
LGTM🧡
I apologize for not explaining the review better...
Thanks for sticking with it!

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**